src/menu/MenuItem.ts

```diff
@@ -0,0 +1,118 @@
+import { assign } from '@dojo/core/lang';
+import { HNode } from '@dojo/widget-core/interfaces';
+import { v } from '@dojo/widget-core/d';
+import ThemeableMixin, { theme, ThemeableProperties } from '@dojo/widget-core/mixins/Themeable';
+import WidgetBase from '@dojo/widget-core/WidgetBase';
+import * as css from './styles/menu.css';
+
+export interface MenuItemProperties extends ThemeableProperties {
+	/**
+	 * Indicates whether the menu is disabled. If true, then the widget will ignore events.
+	 */
+	disabled?: boolean;
+
+	/**
+	 * Applies only when a URL is provided. If `true`, the URL will be opened in a new window.
+	 */
+	external?: boolean;
+
+	/**
+	 * Returns an object of aria properties to apply to the widget's DOM element.
+	 */
+	getAriaProperties?: () => { [key: string]: string; };
```

Comment:
It doesn't, but this will be used in a few different ways requiring different ARIA properties. For example, use in `Menu` requires one set of properties, whereas use with a drop down would require other properties. This seemed like the easiest way to provide any flexibility needed with hard-coding a bunch of scenarios directly within `MenuItem`.